pkg/products/bamboo/helm_release.tf

```diff
@@ -1,6 +1,7 @@
 # Install helm chart for Bamboo Data Center.
 
 resource "helm_release" "bamboo" {
+  depends_on = [kubernetes_job.import_dataset]
```

Comment:
How long does it take to import get completed with given dataset? Any timeout or issue has raised in your test?

Reply:
The import dataset job is ~10 seconds (downloading from s3 and saving the file in shared home). The actual import of the dataset takes around ~30 minutes and I am now changing the timeouts to make it possible. Currently, the `helm_release` will time out but I have increased the timeout but the testing is PITA as the whole cluster needs to be recreated.